.devcontainer/avatax/Dockerfile

```diff
@@ -2,13 +2,10 @@ FROM mcr.microsoft.com/devcontainers/typescript-node:22
 
 ENV PNPM_HOME="/home/node/.pnpm"
 ENV PATH="$PNPM_HOME:$PATH"
-RUN npm install --global corepack@latest
 RUN corepack enable
 
 WORKDIR /app
 
 # This is needed so pnpm-store volume is created with correct permissions (see docker-compose.yml)
 RUN mkdir -p /app/.pnpm-store
 RUN chown -R node:node /app/.pnpm-store
-
-RUN pnpm install --frozen-lockfile --filter=saleor-app-avatax
```

Comment:
do we recommend using this file also without container? if yes, wouldnt it break for others?

Reply:
afaik no - but we can info that user of docker compose needs to run pnpm install & dev on their own